WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -109,22 +112,46 @@ class AuthenticationManager: Authentication {
                                                     LoginPrologueViewController(isFeatureCarouselShown: isFeatureCarouselShown),
                                                 statusBarStyle: .default)
 
+        let getStartedInstructions: String = {
+            if isSimplifiedLoginI1Enabled {
+                return AuthenticationConstants.getStartedInstructionsForSimplifiedLogin
+            } else if isWPComSignupEnabled {
+                return AuthenticationConstants.getStartedInstructionsWithWPComSignupEnabled
+            } else {
+                return AuthenticationConstants.getStartedInstructions
+            }
+        }()
+
+        let continueWithWPButtonTitle: String = {
+            if isSimplifiedLoginI1Enabled {
+                return AuthenticationConstants.loginButtonTitle
+            }
+            return AuthenticationConstants.continueWithWPButtonTitle
+        }()
+
+        let emailAddressPlaceholder: String = {
+            if isSimplifiedLoginI1Enabled {
+                return "name@example.com"
+            }
+            return WordPressAuthenticatorDisplayStrings.defaultStrings.emailAddressPlaceholder
```

Comment:
nit: similar to above, might be simpler:

```suggestion
            return isSimplifiedLoginI1Enabled ? "name@example.com": WordPressAuthenticatorDisplayStrings.defaultStrings.emailAddressPlaceholder
```

Reply:
Updated in 54be2c2f66163406e4146987ebca585357baff3a.